Dockerfile

```diff
@@ -74,71 +65,48 @@ ENV RENOVATE_BINARY_SOURCE=docker
 #============
 FROM final-base as latest
 
-# General tools
-
 # go suggests: git svn bzr mercurial
 RUN install-apt bzr mercurial
```

Comment:
Is there anything preventing us from installing this as part of `install-tool golang`?

Reply:
Yes, they are not strictly dependencies for go. They would only needed when using sources from those repository types